ci-operator/jobs/openshift/machine-api-operator/openshift-machine-api-operator-master-presubmits.yaml

```diff
@@ -69,6 +69,75 @@ presubmits:
         secret:
           secretName: regcred
     trigger: (?m)^/test( | .* )e2e-aws,?($|\s.*)
+  - agent: kubernetes
+    always_run: true
+    branches:
+    - master
+    cluster: api.ci
+    context: ci/prow/e2e-vsphere
```

Comment:
The test we want to run is our machine e2e test suite against an existing IPI cluster.
We need to add to the config https://github.com/openshift/release/blob/037152beab98b4014f1ce7811655e53f477a07f3/ci-operator/config/openshift/machine-api-operator/openshift-machine-api-operator-master.yaml#L55-L60 something the likes of:

```- as: e2e-vsphere-operator
  commands: JUNIT_DIR=${ARTIFACT_DIR} make test-e2e
  openshift_installer_custom_test_image:
    cluster_profile: vsphere
    from: stable:cluster-api-actuator-pkg
    nested_virt_image: ""
```
Then run `make jobs` will generate the jobs for you.

We want this job to be `optional: true`

See for more details https://github.com/openshift/release/blob/master/ci-operator/README.md

Reply:
Then from the mao repo we should be able to run `/test e2e-vsphere-operator`
Is the installer already running e2e against IPI?

Reply:
Seems  we are using steps now
https://github.com/openshift/release/blob/master/ci-operator/config/openshift/installer/openshift-installer-master.yaml#L201-L204

So we need to find the analogous for `openshift_installer_custom_test_image` with steps. See
https://steps.svc.ci.openshift.org/

Reply:
See also https://steps.svc.ci.openshift.org/help/examples

I think we should be able to have something like

```
- as: e2e-vsphere
  steps:
    cluster_profile: vsphere
    workflow: origin-e2e-vsphere
    test:
     From: stable:cluster-api-actuator-pkg
     commands:
        - JUNIT_DIR=${ARTIFACT_DIR} make test-e2e
```